Automattic/wp-calypso
# Update domain only flow: `site-picker` step

## Changes proposed in this Pull Request

This PR enables the reskinned design on domain only flow and updates the `site-picker` step.

It's part of the new domain-only flow (pcYYhz-ye-p2)

![site-picker-before](https://user-images.githubusercontent.com/2797601/150816813-63751698-f31a-44ea-a3e8-2529238e012f.png)

![site-picker-after](https://user-images.githubusercontent.com/2797601/151155420-c6932d53-6ac0-41b5-8516-6ccd971c6837.png)

## Testing instructions

- Build this branch locally or open the live Calypso link
- Navigate to to `/start/domain/domain-only`
- Enter a domain name and select it, in order to move to the next step
- Select `Existing WordPress.com site` on the subsequent step (you need to be logged in to see this option)
- Verify that the layout matches the screenshots above
